packages/http/src/validator/validators/utils.ts

```diff
@@ -79,9 +79,14 @@ export const convertAjvErrors = (errors: NonEmptyArray<ErrorObject>, severity: D
         'additionalProperty' in error.params ? `; found '${error.params.additionalProperty}'` : '';
       const errorPath = error.instancePath.split('/').filter(segment => segment !== '');
       const path = prefix ? [prefix, ...errorPath] : errorPath;
-      const errorPathType = errorPath.length > 0 ? prefix == 'body' ? 'property ' : 'parameter ' : '';
-      const errorSourceDescription = "Request " + (prefix ? `${prefix} ` : '') + errorPathType + errorPath.join('.').trim() + (errorPath.length > 0 ? ' ' : '');
-      
+      const errorPathType = errorPath.length > 0 ? (prefix == 'body' ? 'property ' : 'parameter ') : '';
+      const errorSourceDescription =
+        'Request ' +
+        (prefix ? `${prefix} ` : '') +
+        errorPathType +
+        errorPath.join('.').trim() +
+        (errorPath.length > 0 ? ' ' : '');
+
```

Comment:
Are we missing some setting/config in the Prism codebase that causes this kind of reformatting?

Reply:
No need to do that.  I guess we should just be on the lookout for some formatting/lint-fix option that allows or requires these changes.  Maybe we've got some old files with different formats in that have been "grandfathered" in, or something like that?  I'm just guessing wildly.